### PR TITLE
feat: :sparkles: Add missing soundboard and monetization permissions

### LINKED ISSUE #2216: Soundboard
### Summary

Soundboard

### What is the feature request for?

The core library

### The Problem

Soundboard

### The Ideal Solution

Soundboard

### The Current Solution

No soundboard

### Additional Context

Soundboard

https://github.com/discord/discord-api-docs/pull/6260

### PR DESCRIPTION
**Pull request by Paillat-dev**
## Information

- [ ] This PR fixes an issue. 
  partially fixes but not entirely #2216
Note: permissions were not part of #2321 (now closed), this is additional to that or other pr
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed).
- [ ] This PR is **not** a code change (e.g. documentation, README, typehinting,
      examples, ...).

## Checklist

- [x] I have searched the open pull requests for duplicates.
- [x] If code changes were made then they have been tested.
  - [x] I have updated the documentation to reflect the changes.
- [ ] If `type: ignore` comments were used, a comment is also left explaining why.
- [x] I have updated the changelog to include these changes.

### CODE REVIEW

**Comment by Paillat-dev** [comment]
This:

```py
@bot.slash_command()
async def toggle_soundboard_perms_false(ctx: discord.ApplicationContext, channel: discord.VoiceChannel):
    await channel.edit(overwrites={ctx.guild.default_role: discord.PermissionOverwrite(use_soundboard=False)})
    await ctx.respond(f"Soundboard perms for {channel} have been toggled off.")


@bot.slash_command()
async def toggle_soundboard_perms_true(ctx: discord.ApplicationContext, channel: discord.VoiceChannel):
    await channel.edit(overwrites={ctx.guild.default_role: discord.PermissionOverwrite(use_soundboard=True)})

    await ctx.respond(f"Soundboard perms for {channel} have been toggled on.")

@bot.slash_command()
async def toggle_soundboard_perms_default(ctx: discord.ApplicationContext, channel: discord.VoiceChannel):
    await channel.edit(overwrites={ctx.guild.default_role: discord.PermissionOverwrite(use_soundboard=None)})

    await ctx.respond(f"Soundboard perms for {channel} have been reset to default.")
```

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,6 +30,8 @@ These changes are available on the `master` branch, but have not yet been releas
   ([#2587](https://github.com/Pycord-Development/pycord/pull/2587/))
 - Added optional `filter` parameter to `utils.basic_autocomplete()`.
   ([#2590](https://github.com/Pycord-Development/pycord/pull/2590))
+- Added voice channel permission to use soundboard `discord.Permisions.use_soundboard`
```
Comment (JustaSqu1d) [inline CHANGELOG.md:33]:
```suggestion
- Added voice channel permission to use soundboard `Permissions.use_soundboard`
```

**Review by JustaSqu1d — CHANGES_REQUESTED** [review]

**Comment by JustaSqu1d** [comment]
Might as well also add `USE_EXTERNAL_SOUNDS`(`1 <<45`)

File: discord/permissions.py [hunk 2]
```diff
@@ -610,6 +610,30 @@ def moderate_members(self) -> int:
         """
         return 1 << 40
 
+    @flag_value
+    def view_creator_monetization_analytics(self) -> int:
```
Comment (Lulalaby) [inline discord/permissions.py:614]: u forgot to add it to the slots

**Review by Lulalaby — DISMISSED** [review]

File: discord/permissions.py [hunk 3]
```diff
@@ -762,6 +786,9 @@ class PermissionOverwrite:
         use_external_stickers: bool | None
         start_embedded_activities: bool | None
         moderate_members: bool | None
+        view_creator_monetization_analytics: bool | None
```
Comment (Paillat-dev) [inline discord/permissions.py:789]: @Lulalaby This can't be an overwrite ? Same as `view_guild_insights` for example

**Comment by Paillat-dev** [comment]
Pls don't merge yet

File: discord/permissions.py (hunk 3 above)
Comment (Paillat-dev) [reply discord/permissions.py:789]: Needs to be added to all_channel in that case tho, that I forgot to do.

Comment (Lulalaby) [reply discord/permissions.py:789]: ye add the missing ones

File: CHANGELOG.md [hunk 4]
```diff
@@ -30,6 +30,11 @@ These changes are available on the `master` branch, but have not yet been releas
   ([#2587](https://github.com/Pycord-Development/pycord/pull/2587/))
 - Added optional `filter` parameter to `utils.basic_autocomplete()`.
   ([#2590](https://github.com/Pycord-Development/pycord/pull/2590))
+- Added missing soundboard-related voice channel permissions:
+  `Permissions.use_soundboard`, `Permissions.use_external_sounds`, and
+  [VIEW_CREATOR_MONETIZATION_ANALYTICS](https://discord.com/developers/docs/topics/permissions#:~:text=VIEW_CREATOR_MONETIZATION_ANALYTICS)
+  permission: `Permissions.view_creator_monetization_analytics`
+  ([#2620](https://github.com/Pycord-Development/pycord/pull/2620))
```
Comment (plun1331) [inline CHANGELOG.md:37]: I don't see the point in linking dapi docs for only one of the perms
```suggestion
- Added missing permissions `Permissions.use_soundboard`, `Permissions.use_external_sounds`, and
  `Permissions.view_creator_monetization_analytics`
  ([#2620](https://github.com/Pycord-Development/pycord/pull/2620))
```

**Review by JustaSqu1d — CHANGES_REQUESTED** [review]

**Review by plun1331 — APPROVED** [review]

**Review by plun1331 — APPROVED** [review]

**Review by Dorukyum — DISMISSED** [review]

**Review by Dorukyum — APPROVED** [review]

**Review by plun1331 — APPROVED** [review]

**Pull request merged**